app-backend/batch/src/main/scala/com/azavea/rf/batch/airflow/export/CreateExportDef.scala

```diff
@@ -0,0 +1,165 @@
+package com.azavea.rf.batch.airflow.export
```

Comment:
I don't understand your point; do you mean that `CreateExportDef` should be within the `com.azavea.rf.batch.export` package, along with `Export` & `CommandLine`?